Error when using `format()` on `Node`

It seems like there is something wrong in the `format()` function of node `Node`, where it want's to call `self.type()` which doesn't exist. It seems like it should be `self.kind()` instead.

I am using the example treesitter language "hello" with the zig bindings from https://github.com/tree-sitter-grammars/tree-sitter-zig.

Code: 
```zig
var stdout_buffer: [1024]u8 = undefined;
var stdout_writer = std.fs.File.stdout().writer(&stdout_buffer);
const stdout = &stdout_writer.interface;

const node = tree.?.rootNode();
try node.format(stdout);
```

Error:
```
/redacted/.cache/zig/p/tree_sitter-0.25.0-8heIfwiZAQDptYYd-tp1o_SO4S__lFQwtPgyte9IOVls/src/node.zig:384:21: error: no field or member function named 'type' in 'node.Node'
                self.type(),
                ~~~~^~~~~
/redacted/.cache/zig/p/tree_sitter-0.25.0-8heIfwiZAQDptYYd-tp1o_SO4S__lFQwtPgyte9IOVls/src/node.zig:12:25: note: struct declared here
pub const Node = extern struct {
                 ~~~~~~~^~~~~~
```

zig env:
```zig
.{
    .zig_exe = "/usr/bin/zig",
    .lib_dir = "/usr/lib/zig",
    .std_dir = "/usr/lib/zig/std",
    .global_cache_dir = "redacted",
    .version = "0.15.2",
    .target = "x86_64-linux.6.18...6.18-gnu.2.42",
    .env = .{
        .ZIG_GLOBAL_CACHE_DIR = null,
        .ZIG_LOCAL_CACHE_DIR = null,
        .ZIG_LIB_DIR = null,
        .ZIG_LIBC = null,
        .ZIG_BUILD_RUNNER = null,
        .ZIG_VERBOSE_LINK = null,
        .ZIG_VERBOSE_CC = null,
        .ZIG_BTRFS_WORKAROUND = null,
        .ZIG_DEBUG_CMD = null,
        .CC = null,
        .NO_COLOR = null,
        .CLICOLOR_FORCE = null,
        .XDG_CACHE_HOME = null,
        .HOME = "redacted",
    },
}
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -36,7 +36,6 @@ jobs:
           - x86_64-windows-msvc
           # - aarch64-windows-gnu
           - aarch64-macos-none
-          - x86_64-macos-none
         wasm:
           - "true"
           - "false"
@@ -56,8 +55,6 @@ jobs:
           #   target: aarch64-windows-gnu
           - os: macos-15
             target: aarch64-macos-none
-          - os: macos-13
-            target: x86_64-macos-none
         exclude:
           # doesn't build
           - target: x86_64-windows-msvc

--- a/src/node.zig
+++ b/src/node.zig
@@ -386,10 +386,10 @@ pub const Node = extern struct {
     /// Format the node as a string.
     pub fn format(self: Node, writer: *std.Io.Writer) !void {
         try writer.print(
-            "Node(id=0x{x}, type={s}, start={d}, end={d})",
+            "Node(id=0x{x}, kind={s}, start={d}, end={d})",
             .{
                 @intFromPtr(self.id),
-                self.type(),
+                self.kind(),
                 self.startByte(),
                 self.endByte(),
             },

--- a/src/query_cursor.zig
+++ b/src/query_cursor.zig
@@ -12,7 +12,7 @@ const QueryMatch = extern struct {
         return .{
             .id = self.id,
             .pattern_index = self.pattern_index,
-            .captures = self.captures[0..self.capture_count],
+            .captures = if (self.capture_count == 0) &[_]Query.Capture{} else self.captures[0..self.capture_count],
         };
     }
 };

